test/test_completion_engine.py

```diff
@@ -28,6 +28,15 @@ def test_select_suggests_cols_with_qualified_table_scope():
         Alias(aliases=['tabl']),
         Keyword(last_token='SELECT'))
 
+def test_join_suggests_cols_with_qualified_table_scope():
+    expression = 'SELECT * FROM tabl a JOIN tabl b on a.'
+    suggestions = suggest_type(expression, expression)
+
+    assert suggestions == (
+        Column(tables=((None, 'tabl', 'a'),), drop_unique=None),
+        Table(schema='a'),
+        View(schema='a'),
+        Function(schema='a', filter=None))
```

Comment:
I'm not super confident about this test. Did I do it correctly?

Reply:
Yes, it is correct